webpack.config.js

```diff
@@ -0,0 +1,109 @@
+const path = require('path');
+const { CleanWebpackPlugin } = require('clean-webpack-plugin');
+const CopyWebpackPlugin = require('copy-webpack-plugin');
+const HtmlWebpackPlugin = require('html-webpack-plugin');
+const HtmlWebpackInlineSourcePlugin = require('html-webpack-inline-source-plugin');
+const TerserPlugin = require('terser-webpack-plugin');
+
+module.exports = [{
+  context: path.join(__dirname, 'src'),
+  entry: {
+    './d3-flamegraph': './flamegraph.js',
```

Comment:
Noticed the output of webpack is significantly larger than rollup. ~80%. I was expecting it to be larger, because of the extra webpack magic, but not that much. Wondering if it's importing more files or duplicating imports.

Reply:
`d3-flamegraph.min.js` by rollup is 59 KB, compiled with webpack it is 78 KB (+ 32%). The development version is quite big, but I wouldn't worry about that. The colormapper is bigger (+1 KB), that's because of the webpack header.

About the 59 vs 78 KB: I think rollup is more aggressively deleting dead code. For example I can see the code `schedule = schedules[i]).state > SCHEDULED` of `node_modules/d3-transition/src/active.js` in the webpack version, but not the rollup version. Which looks like unused code.